plugins/filter_kubernetes/kube_conf.h

```diff
@@ -143,6 +143,12 @@ struct flb_kube {
     char *token_file;
     char *token;
     size_t token_len;
+#ifdef FLB_HAVE_KUBE_TOKEN_COMMAND
+    /* Command to get Kubernetes Authorization Token */
+    const char *kube_token_command; 
+
+#endif
```

Comment:
I think you can not put this code behind the CMake flag. Because then you need to do the same for all of the code which uses this field. Otherwise that code won't compile on Windows.